manifests/record.pp

```diff
@@ -29,6 +29,7 @@
   $hash_data,
   $record_type,
   $ensure           = present,
+  $content          = undef,
```

Comment:
Maybe we should set `$content = template('bind/default-record.erb')` and set `$content_template = undef` bellow. But I'm not sure it will work as expected.